### DISCUSSION TITLE
Dexcom, upload follower not working

### DISCUSSION THREAD

**Discussion by jacobbloy** [NightscoutFoundation/xDrip#2981] [Q&A]
Hello, we have our 4yr old daughters Ypsomed connected to camApps FX, Dexcom G6 connected with the pump to a Samsung phone.

I have Xdrip on her phone connected to camps via companion app source and Atm connected to Knightscape to view data on our iPhones.

I'm trying to send data to our dexcom account and to the Iphone follower app.
Iv entered in email and password and nothing, we already have enabled followers and is a click add follower and add the details, it says failed to add follower.

Nothing to do with dexcom shows in the logs of Xdrip.
What am I doing wrong?

**Comment by Navid200** [comment]
You cannot upload to Clarity from xDrip.

As far as uploading to dex share servers, it can easily be interrupted by any updates to how Dexcom handles credentials.

xDrip sync and Nightscout are follower schemes we maintain seriously because they are either 100% open source or have been stable for ever.

If you insist to use dex share, you will need to wait for someone to chime in to help.
But, as far as I know, no active xDrip developer is using it.

**Reply by PieterDeBruijn** [reply]
I would have expected the "Dexcom Share Server upload" would include the upload to Clarity. Is this a difficult addition?

**Comment by corneliusroemer** [comment]
I also struggle to get this set up - interestingly nothing related to upload shows up in the event logs.

Interestingly also, Dexcom Follow app seems to be broken in general - it shows some "temporary server" error.

I also have no idea what the 10-character transmitter serial number is supposed to be - I can't find any 10 character serial number anywhere.

**Reply by PieterDeBruijn** [reply]
Under "Cloud Upload" you can test the login credentials for eg Tidepool, so it would be a nice feature to get a Dexcom Share Server login check.
